### PR TITLE
InsertMulti returning array of IDs

### PR DESCRIPTION
**Pull request by jebentier**
Adding in the ability for insertMulti to retreive an array of IDs that were utilized for the batch insert

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 /vendor/
 /node_modules/
+.idea/
```
Comment (dylanwenzlau) [inline .gitignore]: what is this guy for?

File: src/SQLQuery.php [hunk 2]
```diff
@@ -40,6 +40,7 @@
 	protected $tick;
 	protected $result;
 	protected $return_id = false;
+	protected $insert_multi = false;
```
Comment (dylanwenzlau) [inline src/SQLQuery.php]: let's rename this insert_multi_count = 0 since it's meaning is changed now

File: src/SQLQuery.php [hunk 3]
```diff
@@ -670,6 +672,23 @@ public function insertMultiAssoc(array $data, $ignore = false, $no_escape = fals
 		return $this;
 	}
 
+	/**
+	 * Like insertMultiAssoc(), except it will return an array of "ids" of the row inserted.
```
Comment (dylanwenzlau) [inline src/SQLQuery.php]: rows*

File: .gitignore (hunk 1 above)
Comment (jebentier) [inline .gitignore]: phpstorm project folder

File: src/SQLQuery.php (hunk 2 above)
Comment (jebentier) [inline src/SQLQuery.php]: done d22a35f4d53c376e5eb762da414821673ba95715

File: src/SQLQuery.php (hunk 3 above)
Comment (jebentier) [inline src/SQLQuery.php]: done d22a35f4d53c376e5eb762da414821673ba95715

**Pull request merged**